frontend/src/pages/admin/Template/sections/Environment.vue

```diff
@@ -88,8 +91,8 @@ import FormRow from '@/components/FormRow'
 import FormHeading from '@/components/FormHeading'
 import LockSetting from '../components/LockSetting'
 import ChangeIndicator from '../components/ChangeIndicator'
-import { TrashIcon, PlusSmIcon, LockClosedIcon } from '@heroicons/vue/outline'
-
+import { TrashIcon, PlusSmIcon, LockClosedIcon, CogIcon } from '@heroicons/vue/outline'
```

Comment:
I used cog to differentiate platform defined vars.

Reply:
Would you suggest another icon that indicates it is a platform setting or are happy just to re-use the LockClosedIcon?

Reply:
I suggested we reuse LockClosedIcon because from a users point-of-view it is an env var they cannot edit - so a lock icon would be consistent. I do not think the fact it is system-defined is something that needs indicating.

Reply:
resolved in [f1b77a1](https://github.com/flowforge/flowforge/pull/1039/commits/f1b77a1799f2adfd1e680404ec88144508817fa4)